src/App/Expression.cpp

```diff
@@ -1580,11 +1581,9 @@ void OperatorExpression::_toString(std::ostream &s, bool persistent,int) const
     if (right->priority() < priority()) // Check on operator priority first
         needsParens = true;
     else if (rightOperator == op) { // Equal priority?
-        if (!isRightAssociative())
-            needsParens = true;
-        else if (!isCommutative())
+        if (!isRightAssociative() || !isCommutative())
             needsParens = true;
-    }
+            }
```

Comment:
You probably didn't mean to change this indentation.

Reply:
No , I didn't. Thank you.